react/components/EXPERIMENTAL_Table/README.md

```diff
@@ -642,6 +642,1091 @@ function VisibilityExample() {
+    displayItems: items,
+  })
+
+  const visibility = useTableVisibility({
+    columns,
+    items,
+    hiddenColumns: ['id'],
+  })
+
+  const measures = useTableMeasures({
+    size: state.displayItems.length,
+  })
+
+  const inputSearch = {
+    value: state.inputValue,
+    placeholder: 'Search stuff...',
+    onChange: e => dispatch({ type: 'change', value: e.currentTarget.value }),
+    onClear: () => {
+      dispatch({ type: 'clear' })
+    },
+    onSubmit: e => {
+      e.preventDefault()
+      dispatch({ type: 'submit' })
+    },
+  }
+
+  const buttonColumns = {
+    label: 'Toggle visible fields',
+    showAllLabel: 'Show All',
+    hideAllLabel: 'Hide All',
+    visibility,
+  }
+
+  const density = {
+    label: 'Line density',
+    compactLabel: 'Compact',
+    regularLabel: 'Regular',
+    comfortableLabel: 'Comfortable',
+  }
+
+  const download = {
+    label: 'Export',
+    onClick: () => alert('Clicked EXPORT'),
+  }
+
+  const upload = {
+    label: 'Import',
+    onClick: () => alert('Clicked IMPORT'),
+  }
+
+  const extraActions = {
+    label: 'More options',
+    actions: [
+      {
+        label: 'An action',
+        onClick: () => alert('An action'),
+      },
+      {
+        label: 'Another action',
+        onClick: () => alert('Another action'),
+      },
+      {
+        label: 'A third action',
+        onClick: () => alert('A third action'),
+      },
+    ],
+  }
+
+  const newLine = {
+    label: 'New',
+    onClick: () => alert('handle new line callback'),
+    actions: [
+      'General',
+      'Desktop & Screen Saver',
+      'Dock',
+      'Language & Region',
+    ].map(label => ({
+      label,
+      onClick: () => alert(`Clicked ${label}`),
+    })),
+  }
+
+  const emptyState = {
+    label: 'The table is empty',
+  }
+
+  const empty = React.useMemo(
+    () =>
+      state.displayItems.length === 0 ||
+      Object.keys(visibility.visibleColumns).length === 0,
+    [visibility.visibleColumns, state.displayItems]
+  )
+
+  return (
+    <Table
+      empty={empty}
+      measures={measures}
+      items={state.displayItems}
+      columns={visibility.visibleColumns}
+      emptyState={emptyState}>
+      <Table.Toolbar>
+        <Table.Toolbar.InputSearch {...inputSearch} />
+        <Table.Toolbar.ButtonGroup>
+          <Table.Toolbar.ButtonGroup.Columns {...buttonColumns} />
+          <Table.Toolbar.ButtonGroup.Density density={measures} {...density} />
+          <Table.Toolbar.ButtonGroup.Download {...download} />
+          <Table.Toolbar.ButtonGroup.Upload {...upload} />
+          <Table.Toolbar.ButtonGroup.ExtraActions {...extraActions} />
+          <Table.Toolbar.ButtonGroup.NewLine {...newLine} />
+        </Table.Toolbar.ButtonGroup>
+      </Table.Toolbar>
+    </Table>
+  )
+}
+;<ToolbarExample />
+```
+
+## Totalizer
+
+This uses the Totalizer component between the toolbar and the table content. You can find the full specs on the Totalizer specific docs.
```

Comment:
can we add a link at the totalizer specific docs?

Reply:
Yep, sure